src/main/resources/org/jenkinsci/plugins/workflow/support/actions/WorkspaceRunAction/index.properties

```diff
@@ -0,0 +1,2 @@
+Workspaces=Workspaces
+Workspaces\ for\ =Workspaces for {0}
```

Comment:
nit: Maybe better to switch to a key with no spaces like `WorkspacesFor` to avoid any confusion/complications.